docker.go

```diff
@@ -611,8 +613,8 @@ func WithDefaultBridgeNetwork(bridgeNetworkName string) DockerProviderOption {
 	})
 }
 
-func NewDockerClient() (cli *client.Client, host string, err error) {
-	tcConfig := readTCPropsFile()
+func NewDockerClient() (cli *client.Client, host string, tcConfig TestContainersConfig, err error) {
```

Comment:
We should mark this PR as a breaking change, as we are changing method signature, adding a new returned method. Not a problem though as we are not in 1.0, but at least let's add the label so that it appears as a breaking change in the release notes.

Reply:
Seems that I don't have permissions to set the `type/bc-break` label.

Reply:
Ah no worries, only maintainers add the label (we could probably change it if needed). Will add it if you agree that this is a breaking change.